experiment_buddy/experiment_buddy.py

```diff
@@ -17,10 +17,11 @@
 import wandb
 import wandb.cli
 import yaml
+from funcy import log_durations
 from invoke import UnexpectedExit
 from paramiko.ssh_exception import SSHException
 
-from experiment_buddy.utils import get_backend
+from experiment_buddy.utils import get_backend, remote_time_logger
```

Comment:
import bla
if the line is too long use an alias
if you want to use an alias, don't